src/pages/cartpage/Cart.jsx

```diff
@@ -0,0 +1,26 @@
+import "./Cart.css";
+import { CartProductCard, BillCard } from "../../components";
+const Cart = () => {
+	return (
+		<div>
+			<p className="typo-title flex-hz-center m-1">My Cart</p>
+			<div class="grid grid-2x2">
+				<div class="item-section">
+					<CartProductCard />
+					<CartProductCard />
+					<CartProductCard />
+					<CartProductCard />
+					<CartProductCard />
```

Comment:
suggestion: you can use semantic elements like 'main'  for the dominant content on page instead of 'div'.